packages/sdk/client/src/packlets/devtools/devtools.ts

```diff
@@ -12,7 +12,6 @@ import { Client } from '../client';
  * A hook bound to window.__DXOS__.
  */
 export interface DevtoolsHook {
-  client: Client;
```

Comment:
What the reason for removing this? I'll say that it's been a useful fallback for debugging occasionally, but as devtools stabilizes and expands it's functionality hopefully it should become less and less needed.

Reply:
It is not used anywhere, so I deleted it. Do you mean you were using `window.__DXOS__.client` in the console to debug something?